src/http/api/feed.ts

```diff
@@ -71,7 +71,14 @@ export function createGetFeedHandler(
                 likedByMe: result.post_liked_by_user === 1,
                 replyCount: result.post_reply_count,
                 readingTimeMinutes: result.post_reading_time_minutes,
-                attachments: [], // TODO
+                attachments: result.post_attachments
+                    ? result.post_attachments.map((attachment) => ({
+                          type: attachment.type ?? '',
+                          mediaType: attachment.mediaType ?? '',
+                          name: attachment.name ?? '',
+                          url: attachment.url,
```

Comment:
do we want to `?? ''` on this aswell?

Reply:
Does an attachment without a URL make sense? Should the entity not even allow those?

Reply:
Yeah, attachment.url is a non null field. And we don't need `?? ''` as an attachment without a URL will not make sense.